app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -211,6 +315,8 @@ public static VideoDetailFragment getInstance(int serviceId, String videoUrl, St
         super.onCreate(savedInstanceState);
         setHasOptionsMenu(true);
 
+        activity.setVolumeControlStream(AudioManager.STREAM_MUSIC);
```

Comment:
Default is AudioManager.USE_DEFAULT_STREAM_TYPE. Android version prior to 9 changes ringer volume after pressing on a volume key. After Android 9 the same action triggers music volume change. https://developer.android.com/guide/topics/media-apps/volume-and-earphones  . So STREAM_MUSIC is only needed when a user didn't play anything after entering into the app (if he played something stream type will be STREAM_MUSIC). I removed this line after your comment because I think it's pretty useless here.